### ISSUE TITLE
The effect of tf.train.ExponentialMovingAverage

### ISSUE THREAD

**Issue by BlueWinters** [rinuboney/ladder#16]
I have a little question about the code:
At line 55: 
`ewma = tf.train.ExponentialMovingAverage(decay=0.99)`

My question is:
1. What is the role of the function tf.train.ExponentialMovingAverage? In my option, it may help improve the accuracy of the final classification for mnist? I also read the official doc of tensorflow(https://www.tensorflow.org/api_docs/python/tf/train/ExponentialMovingAverage), but I don't really understand. I guess that the code would also run correctly if there is no tf.train.ExponentialMovingAverage.

I hope some help me to answer this question.
thanks a lot.
